test/metrics/metrics_test.go

```diff
@@ -33,7 +33,16 @@ func TestMetricsWhenUsersDeactivated(t *testing.T) {
 	usersignups := map[string]*toolchainv1alpha1.UserSignup{}
 	for i := 1; i <= 2; i++ {
 		username := fmt.Sprintf("user-%04d", i)
-		usersignups[username] = CreateAndApproveSignup(t, hostAwait, username, member2Await.ClusterName, WithEmail(username+"@redhat.com")) // testing on internal accounts
+
+		// Create UserSignup
+		usersignups[username], _ = NewSignupRequest(t, hostAwait, memberAwait, member2Await).
+			Username(username).
+			Email(username + "@redhat.com").
+			ManuallyApprove().
+			TargetCluster(member2Await).
+			RequireConditions(ConditionSet(Default(), ApprovedByAdmin())...).
```

Comment:
May I know why don't you call `EnsureMUR()`? In this particular case, it could have a negative impact on the tests and could cause test flakiness
```suggestion
			RequireConditions(ConditionSet(Default(), ApprovedByAdmin())...).
			EnsureMUR().
```

Reply:
I still have a number of local changes that I haven't pushed yet due to continued test failures, pretty sure I've added `EnsureMUR()` here already.